en/components/grids_templates/filtering.md

```diff
@@ -382,6 +382,246 @@ You can add a template marked with `igxFilterCellTemplate` in order to retemplat
+    $flat-hover-background: igx-color($dark-palette, "primary", 400),
+    $flat-hover-text-color: igx-color($dark-palette, "secondary", 400)
+);
+```
+
+>[!NOTE]
+>The `igx-color` and `igx-palette` are powerful functions for generating and retrieving colors. Please refer to [`Palettes`](../themes/palette.md) topic for detailed guidance on how to use them.
+
+#### Using Schemas
+
+Going further with the theming engine, you can build a robust and flexible structure that benefits from [**schemas**](../themes/schemas.md). A **schema** is a recipe of a theme.
+
+Extend one of the two predefined schemas, that are provided for every component, in this case - [`light-grid`]({environment:sassApiUrl}/index.html#variable-_light-grid), [`light-input-group`]({environment:sassApiUrl}/index.html#variable-_light-input-group) and [`light-button`]({environment:sassApiUrl}/index.html#variable-_light-button) schemas: 
+
+```scss
+// Extending the light grid schema
+$custom-grid-schema: extend($_light-grid,
+    (
+        filtering-row-text-color:(
+            igx-color: ("primary", 400)
+        ),
+        filtering-row-background:(
+            igx-color: ("secondary", 400)
+        ),
+        filtering-header-text-color:(
+            igx-color: ("primary", 400)
+        ),
+        filtering-header-background:(
+            igx-color: ("secondary", 400)
+        )
+    )
+);
+
+// Extending the light input group schema
+$custom-input-group-schema: extend($_light-input-group,
+    (
+        box-background:(
+            igx-color: ("primary", 400)
+        ),
+        idle-text-color:(
+            igx-color: ("secondary", 400)
+        ),
+        focused-text-color:(
+            igx-color: ("secondary", 400)
+        ),
+        filled-text-color:(
+            igx-color: ("secondary", 400)
+        )
+    )
+);
+
+// Extending the light button schema
+$custom-button-schema: extend($_light-button,
+    (
+        flat-background:(
+            igx-color: ("secondary", 400)
+        ),
+        flat-text-color:(
+            igx-color: ("primary", 400)
+        ),
+        flat-hover-background:(
+            igx-color: ("primary", 400)
+        ),
+        flat-hover-text-color:(
+            igx-color: ("secondary", 400)
+        )
+    )
+);
+```
+
+In order to apply our custom schemas we have to **extend** one of the globals ([`light`]({environment:sassApiUrl}/index.html#variable-light-schema) or [`dark`]({environment:sassApiUrl}/index.html#variable-dark-schema)), which is basically pointing out the components with a custom schema, and after that add it to the respective component themes:
+
+```scss
+// Extending the global light-schema
+$custom-light-schema: extend($light-schema,(
+    igx-grid: $custom-grid-schema,
+    igx-input-group: $custom-input-group-schema,
+    igx-button: $custom-button-schema
+));
+
+// Defining grid-theme with the global light schema
+$custom-grid: igx-grid-theme(
```

Comment:
You can change this to `$dark-grid` for consistency

Reply:
This is on purpose, since in this case we extend the global light schema and using it instead of the dark one.